src/NodaTime.Test/Ignore.cs

```diff
@@ -20,6 +18,16 @@ public static void When(bool condition, string message)
             {
                 Assert.Ignore(message);
             }
-        }        
+        }
+
+        /// <summary>
+        /// Calls Assert.Ignore, which always throws an exception - but with a return type
+        /// of T, which makes it easier to call in expression-bodied members.
+        /// </summary>
+        public static T Throw<T>(string message)
+        {
+            Assert.Ignore(message);
+            throw new InvalidOperationException($"Expected Assert.Ignore to throw");
```

Comment:
Any reason to use an interpolated string here?

Reply:
Nope. Will fix it up when I look at the FIXME I introduced.